OMG.Classroom/frontend/src/App.js

```diff
@@ -1,24 +1,23 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import RegisterPage from './pages/Register';
```

Comment:
Convention - firstly create a components folder in which you will put this pages folder. 

Reason: we can have different components (stateless wrapper components or components for additional React functionality like context API ones, etc.) and it's much better if they are separated into different folders